.env.example

```diff
@@ -0,0 +1,52 @@
+#[Microservices Configuration]
```

Comment:
this file seems to not get packaged properly.  all files that get installed to site-packages need to reside inside one of the package subdirectories.

Reply:
to reproduce the error:
```
python setup.py sdist
pip install dist/*.tar.gz
covalent start --refactor
```

Reply:
should be able to start covalent now